STYLEGUIDE.md

```diff
@@ -51,7 +52,7 @@ Markdown, etc).
     ````
 
 
-### Attributes
+### Attribute Ordering
```

Comment:
I did that because attributes are also discussed elsewhere in the document, and this section is only about ordering.

Actually, there is also a bit of a confusion in that "attribute" is used in two senses in the style guide: for attribute definitions (i.e. `xs:attribute`) and also attributes of tags within the XSD itself.  This section is about the latter, so perhaps an even better name can be used (to distinguish from, say, how `<xs:attribute>` elements should be ordered within a section, which is another topic that has crossed my mind).